openproblems/tasks/denoising/methods/alra.R

```diff
@@ -0,0 +1,15 @@
+library(SingleCellExperiment)
+library(Matrix)
+source("https://raw.githubusercontent.com/KlugerLab/ALRA/master/alra.R")
```

Comment:
is this not something that needs to be installed? it would be good to have some version control to assess which version the method is run in a benchmarking run.